source/audioDucking.py

```diff
@@ -111,8 +111,8 @@ def _ensureDucked():
 		_duckingRefCount += 1
 		if _isDebug():
 			log.debug("Increased ref count, _duckingRefCount=%d" % _duckingRefCount)
+		_setDuckingState(True)
```

Comment:
I think the if check for `_audioDuckingMode != AudioDuckingMode.NONE` is still necessary.